src/borg/archiver.py

```diff
@@ -1984,7 +1984,7 @@ def do_break_lock(self, args, repository):
             any number of characters, '?' matching any single character, '[...]'
             matching any single character specified, including ranges, and '[!...]'
             matching any character not specified. For the purpose of these patterns,
-            the path separator ('\\' for Windows and '/' on other systems) is not
+            the path separator ('\\\\' for Windows and '/' on other systems) is not
```

Comment:
does it also look correct in `borg help` on the shell after that?

Reply:
I don't know; I just sent this from the GitHub web UI. I can try locally if you really want? But it would be a hassle to set up a development environment.

Reply:
yes, please try.

it would not help fixing readthedocs if it breaks the cli help.

Reply:
update: as i suspected, this fix will render "\\" on the shell.

so thanks for finding this, but i'll close this PR as this fix just moves the problem to the cli help.